vuepress/docs/next/tutorials/consume/mt-solr.md

```diff
@@ -0,0 +1,103 @@
+---
```

Comment:
Minor comment, Solr can be enabled with or without multitenancy so it might be better to change the file name and the instructions to note that. Solr is required in a High Availability environment if the application search feature is used.